### PR TITLE
Adding skeleton of Riverwoods file

### PR DESCRIPTION
**Pull request by MikeTrausch**
This more info

### CODE REVIEW

File: MikeTrausch-Riverwoods.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Riverwoods
```
Comment (githubteacher) [inline MikeTrausch-Riverwoods.md:1]: I heard there's a really good Chinese place here!

**Comment by ajskala** [comment]
Terrific!

File: MikeTrausch-Riverwoods.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
```
Comment (RobertSpidale) [inline MikeTrausch-Riverwoods.md:2]: Comment from Bob

File: MikeTrausch-Riverwoods.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+### Places to eat
```
Comment (LTownsDFS) [inline MikeTrausch-Riverwoods.md:3]: Are there actually restaurants in Riverwoods proper? :-)

**Pull request merged**